pkgs/tools/filesystems/mergerfs/tools.nix

```diff
@@ -1,30 +1,36 @@
 { lib, stdenv, fetchFromGitHub, coreutils, makeWrapper
-, rsync, python3, pythonPackages }:
+, rsync, python3Packages }:
 
 stdenv.mkDerivation rec {
   pname = "mergerfs-tools";
-  version = "20190411";
+  version = "20210502";
 
   src = fetchFromGitHub {
     owner = "trapexit";
     repo = pname;
-    rev = "6e41fc5848c7cc4408caea86f3991c8cc2ac85a1";
-    sha256 = "0izswg6bya13scvb37l3gkl7mvi8q7l11p4hp4phdlcwh9jvdzcj";
+    rev = "3b6fe008517aeda715c306eaf4914f6f537da88d";
+    sha256 = "sha256-TQGlvFBYoVO8Qa8rAfIbTgTIsW1I3MfYKT5uwkz175Y=";
   };
 
   nativeBuildInputs = [ makeWrapper ];
-  buildInputs = [ python3 ];
+  buildInputs = [ python3Packages.python ];
 
   makeFlags = [
     "INSTALL=${coreutils}/bin/install"
     "PREFIX=${placeholder "out"}"
   ];
 
-  postInstall = with lib; ''
-    wrapProgram $out/bin/mergerfs.balance --prefix PATH : ${makeBinPath [ rsync ]}
-    wrapProgram $out/bin/mergerfs.dup --prefix PATH : ${makeBinPath [ rsync ]}
-    wrapProgram $out/bin/mergerfs.mktrash --prefix PATH : ${makeBinPath [ pythonPackages.xattr ]}
-  '';
+  postInstall =
+    let
+      binPath = lib.makeBinPath [
+        rsync
+        python3Packages.xattr
+      ];
+    in ''
+      for program in $out/bin/mergerfs.*; do
+        wrapProgram "$program" --prefix PATH: ${binPath}
+      done
+    '';
```

Comment:
```suggestion
  postInstall = ''
      for program in $out/bin/mergerfs.*; do
        wrapProgram "$program" --prefix PATH: ${lib.makeBinPath [ rsync python3.pkgs.xattr ]}
      done
    '';
```

Reply:
tbh, both solutions are absolutely fine. thanks for cleaning this up